### PR TITLE
Use contact phone that better reflects API

### PR DESCRIPTION
**Pull request by tabfugnic**

### CODE REVIEW

File: app/models/icims/candidate.rb [hunk 1]
```diff
@@ -17,11 +17,20 @@ def name
     end
 
     def contact_number
-      phonenumber
+      if phones.present?
```
Comment (JoelQ) [inline app/models/icims/candidate.rb]: Can phones be `nil` ? If it is an empty array, can we get rid of the conditional?

Comment (tabfugnic) [inline app/models/icims/candidate.rb]: It can be nil

Comment (gabebw) [inline app/models/icims/candidate.rb]: What if we make a phones method:

```ruby
def phones
  @phones || []
end
```

Then I think we don't need this `if`?

Comment (tabfugnic) [inline app/models/icims/candidate.rb]: Sounds great to me

**Pull request merged**